index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+<!-- The Links --> 
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="font-awesome/css/font-awesome.css"> 
+</head>
+<body>
+<!-- The Header -->   
+  <header class="header"> 
+  <div class="contect-info" >
+    <span class="time" >12:00</span>  
+    <span class="fa fa-battery-full"></span> 
+    <span class="fa fa-wifi"></span>
+    <span class="fa fa-signal"></span>
+  </div>
+  <div >  
+      <span class="fa fa-bars" id="open"></span>  
+      <h1 class="logo">Gulalai Company</h1> 
+         
+  </div>
```

Comment:
- [ ] The logo is correctly placed but it is a logo, and for convention, semantics, and user experience(UX) reasons it should be wrapped in an `a` tag. Kindly change that. [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
 - [ ] It is missing the hamburger button in the navbar. Please add the hamburger button in a `nav` tag so your header can look like the Figma template. In the future, this button will open a mobile menu so it is very important to have it.
 -Figma template:
<img width="287" alt="image" src="https://user-images.githubusercontent.com/61919617/192573193-2a2bbff4-bb9f-4062-bfb9-8b15a10fe182.png">

Reply:
ok